lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -20,13 +23,29 @@ def include_paths
         end
 
         def languages
-          config.fetch("languages", {})
+          config.fetch("languages", ["ruby"])
         end
 
         def concurrency
           config.fetch("config", {}).fetch("concurrency", 2).to_i
         end
 
+        def checks
+          config.fetch("config", {}).fetch("checks", {})
+        end
+
+        def enabled?(check)
+          checks.fetch(check, {}).fetch("enabled", "false").to_s.casecmp("true").zero?
```

Comment:
1. GTK - will update!
2. I could imagine arguments for either behavior but defaulting to enabled sounds fine.
(makes the specs easier as well :) )